pkg/api/batch.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
```

Comment:
Generally interfaces should be defined in the packages that use them. Is there a reason for this separate API package?

Reply:
I will move everything except for common structs and interfaces: blockchain, cas and protocol clients.

Reply:
I have also removed blockchain and cas interfaces, only protocol stays in api.